workflow/controller/retry_tweak.go

```diff
@@ -0,0 +1,23 @@
+package controller
+
+import (
+	wfv1 "github.com/argoproj/argo/pkg/apis/workflow/v1alpha1"
+	wfutil "github.com/argoproj/argo/workflow/util"
+)
+
+type RetryTweak interface {
```

Comment:
@alexec I was not sure about the interface/implementation, I hope this is what you expected.

Reply:
right direction,  I'm thinking a function:

```golang
type RetryTweak = func(wfv1.RetryStrategy, wfv1.Nodes, *wfv1.Template)
```